build.gradle

```diff
@@ -4,30 +4,28 @@ plugins {
   id "java-gradle-plugin"
   id "java-library"
   id "maven-publish"
-  id "com.github.ben-manes.versions" version "0.43.0"
+  id "com.github.ben-manes.versions" version "0.51.0"
 }
 
 group "com.mx.coppuccino"
 version "3.3.0" // x-release-please-version
-sourceCompatibility = 1.8
-targetCompatibility = 1.8
+sourceCompatibility = JavaVersion.VERSION_17
```

Comment:
You'll definitely need to talk to Matt N about if we're ready for this changed.

Reply:
I wouldn't do this yet.